brave/brave-core
# Lookalike learn more

Resolves brave/brave-browser#31396



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Start with a new profile.
2. Go into `chrome://components`.
3. Trigger an update of the _Safety Tips_ component.
4. Open <http://www.ogogle.com/> in a new tab to trigger this warning:
![ogogle](https://github.com/brave/brave-core/assets/167821/5f4fd0fc-6243-495e-9583-e357eb8fdeb7)
5. Click _Learn more_.
6. Verify that you end up on our [support page for this feature](https://support.brave.com/hc/en-us/articles/17550072876045).
